bookkeeper-server/pom.xml

```diff
@@ -188,7 +188,8 @@
       </extension>
     </extensions>
     <plugins>
-      <plugin>
+    <!--
```

Comment:
what is this for?

Reply:
I left checkstyle plugin configuration commented, I can drop it at all.
It is very useful while testing checkstyle for new API package